client/reviewRegister/addressPage.tsx

```diff
@@ -1,16 +1,21 @@
 import { useState } from 'react';
 import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
-import PostCode from 'components/daum-postcode/review';
-import useReviewRegisterRedux from 'hooks/useReviewRegisterRedux';
-import {dispatchDetailAddress,dispatchPostCodeOpen} from 'store/modules/reviewRegister';
+
+import Map from 'client/reviewRegister/map';
 import Period from 'client/reviewRegister/period';
 import SubHeader from 'client/reviewRegister/subHeader';
-import Map from 'client/reviewRegister/map';
+import PostCode from 'components/daum-postcode/review';
+import useReviewRegisterRedux from 'hooks/useReviewRegisterRedux';
+import {
+  dispatchDetailAddress,
+  dispatchPostCodeOpen,
+} from 'store/modules/reviewRegister';
 
 function AddressRegister() {
   const [state, dispatch] = useReviewRegisterRedux();
+
   const [detailAddress, setDetailAddress] = useState(state.detailAddress);
```

Comment:
Redux가 상태관리를 하는 툴인데
redux에서 상태 관리 중인 ReviewRegisterState의 detailAddress 값을 꺼내서 다시 useState로 상태 관리를 하는 부분이 이상하다고 판단되어 수정 필요
```suggestion
  const dispatch = useDispatch();
  
  const { detailAddress } = useSelector((state: RootState) => state.reviewRegister);
```
또는
```suggestion
  const [{ detailAddress }, dispatch] = useReviewRegisterRedux();
```